indigo_linux_drivers/ccd_gphoto2/indigo_ccd_gphoto2.c

```diff
@@ -1031,15 +1031,25 @@ static int gphoto2_get_key_val(const char *key, char **str,
 static void ctx_error_func(GPContext *context, const char *str, void *data)
 {
 	UNUSED(context);
-	UNUSED(data);
+	if ( data ) {
+		indigo_device *device = *(indigo_device **)(data);
+		if ( device ) {
+			indigo_send_message(device, "ERROR:%s", str);
```

Comment:
you have device in this context plese contruct the message with the device name here!